### ISSUE TITLE
About the inter_class_param for finetuning

### ISSUE THREAD

**Issue by ysc703** [wy1iu/sphereface-plus#1]
Hi wy1iu,

I'm trying to use MHE to finetune my FR model trained with AM-Softmax, as it's mentioned in the paper. 
I have gotten about 0.5% gain at my own dataset with the defalt parameters in the log file(as follow).
  inter_class_param {
    num_output: 10572
    type: AMONG
    iteration: 16000
    alpha_start_iter: 16001
    alpha_start_value: 10
  }
But it seems that the "iteration" and "alpha_start_iter" are too large when finetuning, isn't it? 

Thanks!

**Comment by liulixinkerry** [comment]
Hi @ysc703 ,
Actually, when you set "iteration" as 0 and "alpha_start_iter" as 1, there is no difference in the training process.
You can check the code in [inter_class_layer.cpp#L23](https://github.com/wy1iu/sphereface-plus/blob/master/tools/caffe-sfplus/src/caffe/layers/inter_class_layer.cpp#L23) and [inter_class_layer.cu#L183](https://github.com/wy1iu/sphereface-plus/blob/master/tools/caffe-sfplus/src/caffe/layers/inter_class_layer.cu#L183), you will find these two parameters are just for deciding when (`alpha_start_iter - iteration`) to use MHE. 

eg. if you set `alpha_start_iter:2000` and `iteration:1000`, MHE will be used after 1000 iterations.

**Comment by ysc703** [comment]
Hi @liulixinkerry, 
Thanks for your early reply.
I mistook "iter_" in the code for the train iteration. Now it's clear with your comment.
I'll try again. Thanks!
